### DISCUSSION TITLE
[Feature] Disable the implementation method of adjusting row height, column width and other function points

### DISCUSSION THREAD

**Discussion by Gocgo** [dream-num/univer#3291] [Q&A | 常见问题]
### 初始清单

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 问题

请教下各位老师，是否遇见过同样的问题或者有解决方案，谢谢。
1、univer是否支撑禁用行高、列宽的调整，实现固定行和列的高度；
2、univer是否支撑单元格样式 强制居中展示，新增行和列的时候，新增的行列，也都可以强制居中展示，而不需要人为去点击设置样式（目前有一种方案是监听到变化之后执行sheet.mutation.set-range-values、sheet.commmand.set-style,sheet.command.set-horizontal-text-align三条命令，模拟实现居中指令，但是效率很低）；
3、监听到“退出单元格编辑”动作时，是否可以获取到最近一次有内容更改的单元格坐标；

### 解决方案

希望提供可行的解决方案，万分感谢

[a comment by an automated account is not shown]

**Comment by jikkai** [comment]
Univer supports all of the issues you've raised. We recommend reviewing the documentation for detailed information. 

Given the nature of this issue, we'll be moving it to the Discussions.

**Reply by Gocgo** [reply]
![禁用调整行高](https://github.com/user-attachments/assets/3f6b8259-e899-4f0b-8922-51f265a6cfef)
univer facade提供了setRowHeightsForced接口，可以强制初始化时的行高，但是还是可以通过拖动来调整行高，没有找到禁用行高的接口，请教下是否可以直接禁用调整行高，无法通过拖动来调整行高；
